gravitee-apim-portal-webui-next/src/app/log-out/log-out.component.spec.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { LogOutComponent } from './log-out.component';
+import { AppTestingModule } from '../../testing/app-testing.module';
+
+describe('LogOutComponent', () => {
+  let component: LogOutComponent;
+  let fixture: ComponentFixture<LogOutComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [LogOutComponent, AppTestingModule],
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(LogOutComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
```

Comment:
Maybe add a simple expectLogout which would check what happens in the ngOnInit ? 🤷🏼 
If you judge it's not necessary I would remove this file as nothing is tested.

Reply:
Done